index.html

```diff
@@ -303,7 +303,7 @@
     <p>Unsubscribing works in the same way, except with a single parameter changed to indicate the desire to unsubscribe. Also, the Hub will not validate unsubscription requests with the publisher.</p>
     <section id="subscriber-sends-subscription-request">
       <h3>Subscriber Sends Subscription Request</h3>
-      <p>Subscription is initiated by the subscriber making an HTTPS or HTTP POST [[!RFC7231]] request to the hub URL. This request MUST have a Content-Type header of <samp>application/x-www-form-urlencoded</samp> (described in Section 17.13.4 of [[!W3C.REC-html401-19991224]]) and the following parameters in its body, formatted accordingly:</p>
+      <p>Subscription is initiated by the subscriber making an HTTPS or HTTP POST [[!RFC7231]] request to the hub URL. This request MUST have a Content-Type header of <samp>application/x-www-form-urlencoded</samp> (described in Section 17.13.4 of [[!W3C.REC-html401-19991224]]), using UTF-8 Encoding [[!encoding]], and the following parameters in its body, formatted accordingly:</p>
```

Comment:
Why not reference HTML5 instead of HTML4 for form-url-encoded? Perhaps here: https://www.w3.org/TR/html5/forms.html#application/x-www-form-urlencoded-encoding-algorithm

Instead of:

> ... , using UTF-8 Encoding [[!encoding]], ...

I'd probably say:

> ... `must` use UTF-8 [[!Encoding]] as the document character encoding, ...